src/components/ImageWithSizeCalculation.js

```diff
@@ -63,7 +64,7 @@ class ImageWithSizeCalculation extends PureComponent {
 
             this.props.onMeasure({width, height});
         }, (error) => {
-            console.debug('Unable to fetch image to calculate size', {error});
+            Log.hmmm('Unable to fetch image to calculate size', {error});
```

Comment:
NAB, this would be more useful if this error included the uri for the image we are trying to fetch.

Reply:
Updated to include URI